Entities/Utils/BuildSectionMenu.cs

```diff
@@ -0,0 +1,74 @@
+﻿using GoC.WebTemplate.Components.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GoC.WebTemplate.Components.Utils
+{
+    public static class BuildSectionMenu
```

Comment:
Class names should be a verb like a function/method, a suggest renaming to `SectionMenuBuilder`

Reply:
fixed